enterprise/api/v1/views.py

```diff
@@ -82,6 +110,14 @@ class EnterpriseCustomerUserViewSet(EnterpriseReadOnlyModelViewSet):
     filter_fields = FIELDS
     ordering_fields = FIELDS
 
+    def get_serializer_class(self):
+        """
+        Use a flat serializer for any requests that aren't read-only.
+        """
+        if self.request.method in ('GET', ):
+            return serializers.EnterpriseCustomerUserSerializer
```

Comment:
We've been specific about Read vs. Write in other areas -- do we need to follow-suit with this serializer as well and name it `EnterpriseCustomerUserReadSerializer` (or ReadOnlySerializer)?